The tool latexmk is popular enough to justify having it in the default
installation and historically been installed when just installing base
texlive. Hence, move it from collection-binextra to collection-basic.
--- a/texlive-2025.0.tlpdb	2024-06-16 17:59:57.211022910 +0200
+++ b/texlive-2025.0.tlpdb	2024-06-16 18:01:31.523591755 +0200
@@ -64801,6 +64801,7 @@ depend texlive.infra
 depend tlshell
 depend unicode-data
 depend xdvi
+depend latexmk
 
 name collection-bibtexextra
 category Collection
@@ -65033,7 +65034,6 @@ depend latex2nemeth
 depend latexdiff
 depend latexfileversion
 depend latexindent
-depend latexmk
 depend latexpand
 depend light-latex-make
 depend listings-ext
